### ISSUE TITLE
Nowhere by Radix (Protracker) sounds off

### ISSUE THREAD

**Issue by blaqmajik** [steffest/BassoonTracker#21]
First of all: great project! 

I took it for a test drive with a few tunes and noticed it is not quite there yet for the tune mentioned above. The same probably goes for some of WOTW's intro tunes which use very similar sounds and a similar technique, where this tune has borrowed from.

Available here: https://modland.com/pub/modules/Protracker/Radix/nowhere.mod

Check out the playback on the real thing or with Bubsy's Protracker clone, comes a lot closer.

**Comment by steffest** [comment]
Confirmed.
It caused by the "EFx" command not being implemented.
That's the "invert loop" command.
I kind of forgot about that.
It's also not really clear exactly how this command works.
The consensus  seems to be that it "DESTROYS" and it "RULEZZZZzzz"

**Comment by blaqmajik** [comment]
> The consensus seems to be that it "DESTROYS" and it "RULEZZZZzzz"

It definitely does! (RULE!)

**Comment by 8bitbubsy** [comment]
Yes, this effect is controversial because it actually trashes the sample data.
